### ISSUE TITLE
[SCREENREADER]: GIBCT EYB - JAWS and IE11 SHOULD NOT announce extra information when I toggle accordions

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10200]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

JAWS and IE11 read out a lot of extra information when users expand/collapse the benefits filters or the larger gray accordions. I've heard this off and on, but could never pinpoint it. I think it has to do with the `aria-hidden="true | false"` attributes on the accordions. If we can prevent JAWS from reading this extra information, it would be a big boost for this subset of users.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] JAWS + IE11 does not read out extra information when I toggle the accordions open or closed
- [ ] Updated page throws no axe errors

## Environment

* Windows 10
* IE11
* JAWS 2019
* https://staging.va.gov/gi-bill-comparison-tool/profile/11929140 (this is just a sample, the behavior is exhibited on all profile views)

## Steps to Recreate

1. Start JAWS
1. Enter a GIBCT profile URL and load the page
1. Tab or arrow to the Estimate your benefits accordion
1. Tab to the first sub-accordion, Your military details
1. Press `SPACE` to close the accordion. Verify the paragraph above it is read aloud. This is not the expected behavior. Nothing should be read aloud at this point.

## Possible Fixes (optional)
All of the accordion DIVs have an `aria-hidden` attribute that is being toggled true or false. I have a feeling this might be causing the issue. Could we consider updating this behavior to use the `hidden` attribute instead?

```diff
/* The hidden attribute would appear when the div is closed */
<div 
- aria-hidden="true"
  class="section-content"
  id="accordion-item-12"
+ hidden
>
```

## WCAG or Vendor Guidance (optional)

* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/13145

**Comment by delli-gatti-michael** [comment]
local testing complete

**Comment by delli-gatti-michael** [comment]
@1Copenut The PR for this is merged and the changes are now in Staging

**Comment by desireeturner** [comment]
@Clark-W-Griswold this is ready for review on Staging

**Comment by 1Copenut** [comment]
@delli-gatti-michael I reviewed this on Staging using IE11 + JAWS for profile ID `32002927` Bryan College of Health Sciences. The issue is still happening when I expand/collapse the sub accordions in Estimate your benefits.

**Comment by zurbergram** [comment]
@1Copenut with IE11+JAWS i am unable to reproduce either behavior mentioned in https://github.com/department-of-veterans-affairs/va.gov-team/issues/10200#issuecomment-652056267

**Comment by 1Copenut** [comment]
@zurbergram I was able to re-create this using IE11 + JAWS 2019 this morning. I figured out a way to capture it to video, but the file is pretty large (47MB). I can upload it to a drive and share the link or we can review on a call if needed.

I took another look at the source code and noticed a small aria-live region at the bottom of the page. I'm unsure what it does, but it's the last idea I have:

```html
<div id="a11y-status-message" role="status" aria-live="assertive" aria-relevant="additions text" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; width: 1px;"></div>
```

Do you know what this does or if it's required for page functionality?

/cc @KevinMHoffmanUSDS

**Comment by ksantiagomorgan6** [comment]
Pull request: https://github.com/department-of-veterans-affairs/vets-website/pull/13486

**Comment by sonntag-adam** [comment]
@1Copenut in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam JAWS 2019 is still reading the paragraph above the sub-accordions sometimes. It's not consistently doing that, and it's helpful in some cases, so I'm going to close this issue. The VA 508 might flag it as something to give a second look, but I'll let them make the final call.
